### PR TITLE
doc(Showcase): all components with next.js

### PR DESCRIPTION
**Pull request by frassinier**
**What is the problem this PR is trying to solve?**
Next.js based showcase was not complete 

**What is the chosen solution to this problem?**
Complete with more and more examples

**Please check if the PR fulfills these requirements**

- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -1,43 +1,395 @@
...
+						withBackground
+					/>
+				</section>
+				<section>
+					<h2>Links</h2>
+					<Link href="#">Link</Link>
+				</section>
+				<section>
+					<h2>Layout</h2>
+					<h3>Stack</h3>
```
Comment (jsomsanith-tlnd) [inline pages/index.tsx:312]: What is this h3 ?

File: pages/index.tsx [hunk 2]
```diff
@@ -1,43 +1,395 @@
...
+				</section>
+				<section>
+					<h2>Tooltips</h2>
+					<Tooltip title="tooltip title">
+						<Button>Button</Button>
+					</Tooltip>
+				</section>
+				<section>
+					<h2>Visually Hidden</h2>
+					<VisuallyHidden>this content is visually hidden</VisuallyHidden>
```
Comment (jsomsanith-tlnd) [inline pages/index.tsx:390]: This doesn't show anything in the showcase :D

Comment (frassinier) [reply pages/index.tsx:390]: WDYT?
```suggestion
					This content is visible<VisuallyHidden> and, this, is visually hidden</VisuallyHidden>!
```

File: pages/index.tsx (hunk 1 above)
Comment (frassinier) [reply pages/index.tsx:312]: To be able to increase the list

**Review by jsomsanith-tlnd — APPROVED** [review]

**Pull request merged**